filebeat/docs/reload-configuration.asciidoc

```diff
@@ -47,6 +62,50 @@ definitions. For example:
   scan_frequency: 5s
 ------------------------------------------------------------------------------
 
+
 WARNING: It is critical that two running prospectors DO NOT have overlapping
 file paths defined. If more than one prospector harvests the same file at the
 same time, it can lead to unexpected behaviour.
+
+[float]
+[[reload-module-config]]
+=== Module config
+
+For module configurations, you specify reload options in the
+`filebeat.config.modules` section of the +{beatname_lc}.yml+ file. For example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+filebeat.config.modules:
+  enabled: true
+  path: configs/*.yml
+  reload.enabled: true
+  reload.period: 10s
+------------------------------------------------------------------------------
+
+TIP: If you are using the <<modules-command,`modules`>> command to enable and
+disable configurations, you can enable config reloading for the `modules.d`
+directory by specifying `path: ${path.config}/modules.d/*.yml`.
+
+Each file found by the Glob must contain a list of one or more module
+definitions. For example:
+
+//REVIEWERS: I got an error when I tried to have an empty line before the -module: mysql entry below. Shouldn't whitespace be allowed there?
```

Comment:
Weird. I was not able to reproduce this issue. Maybe I had a tab or something on that line? Anyhow...if I can't reproduce the problem, it doesn't exist, right? :-)